java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/Application.java

```diff
@@ -50,16 +51,37 @@
   private static final Options CLI_OPTIONS = new Options();
   private static final String EXECUTABLE_NAME = "<BUILDER>";
 
+  public static final ImmutableMap<String, String> DEFAULT_JDK_MAPPINGS = ImmutableMap.of(
+      "*", "gcr.io/google-appengine/openjdk:8",
+      "openjdk8", "gcr.io/google-appengine/openjdk:8",
+      "openjdk9", "gcr.io/google-appengine/openjdk:9"
+
+  );
+  public static final ImmutableMap<String, String> DEFAULT_SERVER_MAPPINGS;
```

Comment:
Why don't you use `ImmutableMap.of` here as well as above?

Reply:
It's too many arguments :(  `of` is only defined for up to like 8 items.

Reply:
I see. Have you tried `ImmutableMap.builder().put(...)`?